### PR TITLE
feat: add Keto config

### PR DESCRIPTION
**Pull request by zepatrik**

### CODE REVIEW

File: cmd/cloudx/testhelpers/testmain.go [hunk 1]
```diff
@@ -1,23 +1,48 @@
...
 )
 
-func RunAgainstStaging(m *testing.M) {
-	// Run the tests
-	result := m.Run()
-
-	// Use staging
-	if err := os.Setenv("ORY_CLOUD_CONSOLE_URL",
-		stringsx.Coalesce(os.Getenv("ORY_CLOUD_CONSOLE_URL"),
-			"https://project.console.staging.ory.dev")); err != nil {
```
Comment (zepatrik) [inline cmd/cloudx/testhelpers/testmain.go:17]: Previously we first ran the test, and then swichted to staging. So of course everything ran against prod :sweat_smile:

**Review by aeneasr — COMMENTED** [review]
Should we add `update permission-config` as well?

File: cmd/cloudx/testhelpers/testmain.go (hunk 1 above)
Comment (aeneasr) [reply cmd/cloudx/testhelpers/testmain.go:17]: whoops

**Review by zepatrik — COMMENTED** [review]
> Should we add update permission-config as well?

Yes, somehow missed it :sweat_smile:

File: cmd/cloudx/testhelpers/testmain.go [hunk 2]
```diff
@@ -1,23 +1,48 @@
...
+func UseStaiging() {
```
Comment (aeneasr) [inline cmd/cloudx/testhelpers/testmain.go:11]:
```suggestion
func UseStaging() {
```

**Review by aeneasr — COMMENTED** [review]
Just a minor thing but basically mergable

File: cmd/cloudx/project/get_test.go [hunk 3]
```diff
@@ -16,43 +15,32 @@ import (
 )
 
 func TestGetProject(t *testing.T) {
-	configDir := testhelpers.NewConfigDir(t)
-	cmd := testhelpers.ConfigAwareCmd(configDir)
-	email, password := testhelpers.RegisterAccount(t, configDir)
-
-	project := testhelpers.CreateProject(t, configDir)
-
 	t.Run(fmt.Sprintf("is able to get project"), func(t *testing.T) {
```
Comment (aeneasr) [inline cmd/cloudx/project/get_test.go:18]: Looks like you removed tests for `get identity-config` and did not add tests for `get permission-config`. Is that on purpose or accident?

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: cmd/cloudx/project/get_test.go (hunk 3 above)
Comment (zepatrik) [reply cmd/cloudx/project/get_test.go:18]: Some tests were moved to `cloudx/client`, but yeah I'll have to add  tests for permission config.

**Pull request merged**